CONTRIBUTING.MD

```
@@ -0,0 +1,153 @@
+
+---
+
+## Table of Contents
+
+1. [Tech Stacks Used](#-tech-stack)
+    - [Frontend](#frontend)
+    - [AI & Computer Vision](#ai--computer-vision)
+    - [Backend & Database](#backend--database)
+    - [Data Processing](#data-processing)
+    - [UI/UX](#uiux)
+2. [Prerequisites](#-prerequisites)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)